doc/input-output-reference/src/overview/group-zone-forced-air-units.tex

```diff
@@ -4024,6 +4024,10 @@ \subsubsection{Inputs}\label{inputs-14-018}
 
 This numeric field defines the maximum outdoor dry-bulb temperature in degrees Celsius for this VRF terminal unit supplemental heating coil operation. The supplemental heater will not operate when the outdoor dry-bulb temperature is above this value. The maximum value must be less than or equal to 21°C. If this field is left blank, the default value of 21°C will be used.
 
+\paragraph{Field: Controlling Zone or Thermostat Location}\label{controlling-zone-or-thermostat-location-2-110}
+
+This alpha field is used only for air loop equipment and defines the controlling zone or thermostat location for load based operational control. This field is not required for zone equipment or equipment used in outdoor air systems. Leave this field blank for zone equipment, outdoor air equipment and air loop equipment that is terminal unit or coil air outlet controlled based on a set point termperature. When the terminal unit is used in an air loop and this terminal unit is load controlled, this zone's thermostat will control operation.
+
```

Comment:
I would rephrase this: "This alpha field is used only used *when this VRF terminal unit is used as air loop equipment with load based operational control. *" And the rest is a bit circular. I would move the last sentence up sooner - talk first about load based operation. Then mention set point based operation, and finally say it's ignored for zone equipment and oa equipment.

More than this, the rest of the I/O Ref for this object needs modifications for these new applications - the overview paragraph, the fan inputs, and the OA mixer inputs.